docs/intro/install.mdx

```diff
@@ -4,10 +4,6 @@ description: Learn how to install and get running with Helm.
 sidebar_position: 2
 ---
 
-import Helm4 from "../_v4-in-progress.mdx" 
-
-<Helm4/>
```

Comment:
before removing this line there is one more mention under "from source" about the "latest (pre-release)" of Helm. you'll want to remove the pre-release parentheses now.